pkg/mflag/flag.go

```diff
@@ -1086,7 +1095,6 @@ func (cmd *FlagSet) ReportError(str string, withHelp bool) {
 		}
 	}
 	fmt.Fprintf(cmd.Out(), "docker: %s.\n", str)
-	os.Exit(1)
```

Comment:
I see, you added an exit(1) above, but there are other calls to ReportError- like in CmdCreate - won't those break?